docs/source/blocks/listing-block.md

```diff
@@ -0,0 +1,60 @@
+---
+myst:
+  html_meta:
+    'description': "The listing block provides an additional extension letting the developer modify the default 'No results found' message using the configuration of the block"
+    'property=og:description': "The listing block provides an additional extension letting the developer modify the default 'No results found' message using the configuration of the block"
+    'property=og:title': 'Listing block extensions mechanism'
+    'keywords': 'Volto, Plone, frontend, React, Upgrade, Guide, Block extensions, variations, schema enhancers, listing block'
```

Comment:
Please use `"` instead of `'` for all values. This is a default Python lint configuration.